README.md

```diff
@@ -106,7 +106,7 @@ Setup for developers (Windows)
  
      ```bash
      $ virtualenv venv
-     $ ./venv/Scripts/activate
+     $ .\venv\Scripts\activate
```

Comment:
Do you not need a `.bat` after the extension?

Reply:
i think in CMD `.\venv\Scripts\activate` and `.\venv\Scripts\activate.bat` both can work. Problem arises when we are working on windows powershell.

Reply:
as in windows powershell we can not use `.\venv\Scripts\activate.bat`